src/components/API/Api.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from "react";
+import AppleIcon from '@mui/icons-material/Apple';
+import axios from "axios";
+
+export default function Api() {
+  useEffect(() => {
+    async function ApiData() {
```

Comment:
there's a better way to do this, "Self calling function"
```suggestion
    (async(){
    ...your function code
    })()
```
Link to documentation: https://scriptverse.academy/tutorials/js-self-invoking-functions.html#:~:text=A%20self%2Dinvoking%20(also%20called,invoked%20immediately%20after%20its%20definition.&text=A%20self%2Dinvoking%20function%20can,be%20passed%20as%20a%20parameter.